### PR TITLE
Update sidebar UI

### PR DESCRIPTION
**Pull request by Jesus89**
Related to #12479

Changes in the sidebar style:
- Update Pencil icon: icon-font_108_pencilMenu
- Update Marker's proportion: 11/32
- Update Marker's top to 20px
- Update Pencil margin to 8px

Extra changes:
- Update Pencil icon size: 16x16px
- Update Marker-Pencil icon distance: 48px
- Update sidebar color: https://github.com/CartoDB/CartoAssets/pull/169

### CODE REVIEW

**Comment by rubenmoya** [comment]
Looks good, let's go to acceptance!

**Comment by rubenmoya** [comment]
It looks good to me, but @CartoDB/design have the last word, it's deployed in `ded01`

File: app/assets/stylesheets/editor-3/_menu.scss [hunk 1]
```diff
@@ -125,7 +125,8 @@
     height: 24px;
   }
   .EditorMenu-navigationIcon {
-    font-size: 14px;
+    margin: 4px;
+    font-size: 12px;
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/_menu.scss]: I think we have a variable for this.

Comment (nobuti) [inline app/assets/stylesheets/editor-3/_menu.scss]: https://github.com/CartoDB/CartoAssets/blob/master/src/scss/cdb-variables/_sizes.scss#L18

Comment (Jesus89) [inline app/assets/stylesheets/editor-3/_menu.scss]: OK. Free to change!

**Comment by josecruz** [comment]
Taking a look 
👀 👀

**Comment by josecruz** [comment]
Pretty close guys, nice jobs! Just found some subtle differences:

1. Sidebar icons are at 16x16px
2. Sidebar separation between marker and icon is 48px
3. Sidebar color is: #1785FB (yep, we were using a wrong blue)

**Comment by rjimenezda** [comment]
Deploying to ded04, will mention someone from design when it's there

**Comment by rjimenezda** [comment]
@CartoDB/design Can someone do the acceptance? It's on ded04

**Comment by josecruz** [comment]
@rjimenezda sorry! I take care :D

**Comment by josecruz** [comment]
Color and size change looking good! The only thing I'm missing is the separation between the marker and the first icon which should be 48px (right now is 64px)

**Comment by Jesus89** [comment]
The shorter distance between the elements is 48px (48 = 64 - 32 / 2)

**Comment by Jesus89** [comment]
Deployed to ded01. @josecruz can you check that all is OK?

**Comment by josecruz** [comment]
This is pixel-perfect. Amazing job guys, thanks!

💖

**Pull request merged**